tools/benchmark/cmd/watch_latency.go

```diff
@@ -61,31 +61,23 @@ func init() {
 }
 
 func watchLatencyFunc(_ *cobra.Command, _ []string) {
-	key := string(mustRandBytes(watchLKeySize))
+	key := "/registry/pods"
```

Comment:
How this impacts the results? If it doesn't then please remove.

Reply:
It does not. Will remove. 

Just to demonstrate that it could happen on pods, and it's a generic problem (not limited to events)